templates/conf.d/tcp_check.yaml.j2

```diff
@@ -0,0 +1,14 @@
+---
+
+init_config:
+
+instances:
+{% for i in datadog.integrations.tcp_check %}
+
+  - name: {{ i.name }}
+    host: {{ i.endpoint }}
+    port: {{ i.port }}
+    timeout: 1
+    collect_response_time: true
```

Comment:
the same here - default to `true`, but allow to set to something else when required.